src/main/java/seedu/address/model/note/Note.java

```diff
@@ -2,15 +2,20 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.HashSet;
 import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
 
 
 /**
  * Generic Note class for country and client notes.
  */
 public class Note {
-    public static final String MESSAGE_CONSTRAINTS = "message contraints for note";
+    public static final String MESSAGE_CONSTRAINTS = "message constraints for note";
```

Comment:
Followed your suggestion. Will test using the GUI instead of just using automated tests so that I don't overlook such things henceforth. 

As usual, I'm amazed by your sharp eye in catching these things 😃